ansible_collection/hpe/nimble/CHANGELOG.rst

```diff
@@ -0,0 +1,65 @@
+=================================================
+HPE Nimble Storage Ansible Modules Release Notes
+=================================================
+
+.. contents:: Topics
+
+v1.0.2
+======
+
+Release Summary
+---------------
+
+This release provides compatibilty for Ansible v4.0.0
+
+Minor Changes
+-------------
+
+- Made it compatible for Ansible v4.0.0
+
+v1.0.1
+======
+
+Release Summary
+---------------
+
+This release provides compatibilty for Ansible v3.0.0
+
+Minor Changes
+-------------
+
+- Made it compatible for Ansible v3.0.0
+
+v1.0.0
+======
+
+Release Summary
+---------------
+
+First release of the `HPE Nimble Storage Content Collection for Ansible`.
+
+New Modules
```

Comment:
No folder still?!

Reply:
yes as we discussed this release is still with no code changes.. implementing folder will take time and we will surely miss the deadline. plan was to take it in next release.